src/openrct2/ride/ride.h

```diff
@@ -759,7 +759,7 @@ enum {
 	RIDE_TYPE_FLAG_HAS_DATA_LOGGING = 1 << 9,
 	RIDE_TYPE_FLAG_HAS_DROPS = 1 << 10,
 	RIDE_TYPE_FLAG_NO_TEST_MODE = 1 << 11,
-	RIDE_TYPE_FLAG_TRACK_ELEMENTS_HAVE_TWO_VARIETIES = 1 << 12,	// used by rides with two variaties, like the u and o shapes of the dinghy slide and the dry and submerged track of the water coaster
+	RIDE_TYPE_FLAG_TRACK_ELEMENTS_HAVE_TWO_VARIETIES = 1 << 12,	// used by rides with two variates, like the u and o shapes of the dinghy slide and the dry and submerged track of the water coaster
```

Comment:
I need to fix this lol, you're correct. I clicked the wrong word when making the correction. Thank you.